obs-studio-server/source/nodeobs_service.cpp

```diff
@@ -2122,16 +1979,11 @@ void OBS_service::setReplayBufferOutput(obs_output_t* output)
 	replayBufferOutput = output;
 }
 
-void OBS_service::updateStreamSettings(void)
+void OBS_service::updateStreamingOutput(bool isSimpleMode)
 {
 	const char* currentOutputMode = config_get_string(ConfigManager::getInstance().getBasic(), "Output", "Mode");
```

Comment:
We are passing `isSimpleMode` to this function but it is never used. Instead of getting the output mode again it is better to use `isSimpleMode` in the `if` right below this

Reply:
Fixed 👍